lib/ioc-rc

```diff
@@ -555,7 +557,8 @@ __rc_jails () {
         local boot_order=$(sort -n $boot_list)
         local shutdown_order=$(sort -rn $boot_list)
     else
-        __info "no jails found with property boot=on, exiting.."
+        __info "no jails found with property boot=on or
+        with jail_hostid: $hostid on this host, exiting.."
```

Comment:
I hope it is ok now.
I do not know how to get rid of the other commit messages!
They where try outs but still going with the pull request.




Op 15/09/15 om 00:16 schreef Brandon Schneider:
>
> In lib/ioc-rc
> <https://github.com/iocage/iocage/pull/222#discussion_r39454503>:
>
> > @@ -555,7 +557,8 @@ __rc_jails () {
> >          local boot_order=$(sort -n $boot_list)
> >          local shutdown_order=$(sort -rn $boot_list)
> >      else
> > -        __info "no jails found with property boot=on, exiting.."
> > +        __info "no jails found with property boot=on or
> > +        with jail_hostid: $hostid on this host, exiting.."
>
> Sorry I wasn't thinking straight. It's been "that" kind of a Monday.
> Change line 560 to:
>
> |__info "no jails found with property boot=on or |
>
> and line 561 to:
>
> |echo " with jail_hostid: $hostid on this host, exiting." |
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/iocage/iocage/pull/222/files#r39454503>.
>